tests/query_conf.cpp

```diff
@@ -695,6 +695,19 @@ TEST(query_repository_make_query, should_return_query_for_struct_parameters_pass
     );
 }
 
+TEST(query_repository_make_query, should_return_query_for_struct_parameters_passed_by_reference) {
+    const auto repository = ozo::make_query_repository(
+        "-- name: query with struct parameters\n"
+        "SELECT :string::text || :number::text",
+        hana::tuple<query_with_struct_parameters>()
+    );
+    struct_parameters parameters {std::string_view("42"), 13};
+    EXPECT_EQ(
+        repository.make_query<query_with_struct_parameters>(parameters),
```

Comment:
It would be nice to point reference explicitly:

```suggestion
        repository.make_query<query_with_struct_parameters>(static_cast<struct_parameters&>(parameters)),
```